spec.html

```diff
@@ -33492,11 +33492,11 @@ <h1>String.fromCharCode ( ..._codeUnits_ )</h1>
         <p>This function may be called with any number of arguments which form the rest parameter _codeUnits_.</p>
         <p>It performs the following steps when called:</p>
         <emu-alg>
-          1. Let _elements_ be a new empty List.
+          1. Let _result_ be the empty String.
           1. For each element _next_ of _codeUnits_, do
-            1. Let _nextCU_ be ℝ(? ToUint16(_next_)).
-            1. Append _nextCU_ to _elements_.
-          1. Return the String value whose code units are the elements in the List _elements_. If _codeUnits_ is empty, the empty String is returned.
+            1. Let _nextCU_ be the code unit whose numeric value is ℝ(? ToUint16(_next_)).
+            1. Append _nextCU_ to _result_.
```

Comment:
I'm a little suspicious of this usage of "append". It's supposed to update _the alias_ `result`, rather than mutating the non-mutable String value which the alias refers to. But we use exactly the same syntax - "append v to list" - to _mutate_ lists.

Reply:
I agree; append means mutate, not “concatenate and reassign”

Reply:
Oops, you're right.